authentication/azure/auth.go

```diff
@@ -63,6 +63,9 @@ func NewEnvironmentSettings(resourceName string, values map[string]string) (Envi
 		// The resource name to request a token is https://eventhubs.azure.net/, and it's the same for all clouds/tenants.
 		// Kafka connection does not factor in here.
 		es.Resource = "https://eventhubs.azure.net"
+	case "signalr":
+		// Azure SignalR (data plane)
+		es.Resource = "https://signalr.azure.com"
```

Comment:
Is SignalR available in other clouds and does this identifier need to change accordingly?